test/e2e/docs/test_environment.md

```diff
@@ -29,7 +29,7 @@ export CONFIG_KEY=<decryption_key_from_a8c_store>
 Optional:
 
 ```
-export SAVE_AUTH_COOKIES=<true/false>
+export AUTHENTICATE_ACCOUNTS=p2User,i18nUser
```

Comment:
Neat. Do we have any tests that actually use this pre-authentication feature for multiple accoutns?

Reply:
The accounts we pre-authenticate have been set to `simpleSitePersonalPlanUser`, `eCommerceUser`, and `defaultUser`. Before this PR, they were defined in `globalSetup.js`, and with this change, they're going to be passed with this env var. So it's not a new feature, just an existing one that got a bit refactored.